File: JavaScript/1177-PreenchimentoDeVetorII.js
var input = require("fs").readFileSync('stdin', 'utf8');
var lines = input.split('\n');
var t = parseInt(lines.shift());
var c = 0;
var n = [];
for(var i = 0; i < 1000; i++){
    if(c<t){
        console.log(`N[${i}] = ${c}`);
        c++;
    } else {
        c = 0;
        console.log(`N[${i}] = ${c}`);
        c++;
    }
}

File: Java/Python/1177-PreenchimentoDeVetorII.py
n = []
c = 0
t = int(input())
for i in range(0,1000):
    if c < t:
        print(f'N[{i}] = {c}')
        c += 1
    else:
        c = 0
        print(f'N[{i}] = {c}')
        c += 1
